docs/release/Code-signing-metadata.md

```diff
@@ -185,12 +185,12 @@ that exist in a zip bundle called `ios/artifacts.zip`.
     `ios_file_without_entitlements` variable.
 
 You can reference the generator script
-[create_full_ios_framework.py](https://github.com/flutter/engine/blob/main/sky/tools/create_full_ios_framework.py).
+[create_ios_framework.py](https://github.com/flutter/engine/blob/main/sky/tools/create_ios_framework.py).
 
 ## Code signing artifacts other than flutter engine binaries
 
 The code signing functionality is implemented as [a recipe module in flutter recipes](https://cs.opensource.google/flutter/recipes/+/master:recipe_modules/signing/api.py). Therefore it can also be used to
 code sign arbitrary flutter artifacts built through recipe, for example, flutter iOS usb dependencies.
 
 To code sign, after the artifacts are built, pass the file paths into
-the code signing recipe module and invoke the function. An example is [how engine V2 invokes the code signing recipe module](https://cs.opensource.google/flutter/recipes/+/master:recipes/engine_v2/engine_v2.py;l=197-212).
\ No newline at end of file
+the code signing recipe module and invoke the function. An example is [how engine V2 invokes the code signing recipe module](https://cs.opensource.google/flutter/recipes/+/master:recipes/engine_v2/engine_v2.py;l=197-212).
```

Comment:
The diff is that this line had no trailing newline, and as we all know, POSIX therefore says [it's no line at all](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206).

Well, that, and my editor added it automatically without me asking.